packages/ai-rules/.ruler/data-modeling/casting_dbt_staging_models.md

```
@@ -0,0 +1,16 @@
+# Casting data-types for staging models
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)